.github/ISSUE_TEMPLATE/security_bug.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Security Bug report'
+about: |
+    Don't use the issue tracker for this!!1! Please write an e-mail to
```

Comment:
1?

Reply:
Yes!!1! (I can remove it though if you think it is *too* emphasized)

Reply:
(https://www.urbandictionary.com/define.php?term=%21%21%211%2111%21%21%21oneoneeleven)

Reply:
Those insider jokes are not helpful. Just remove.

Reply:
Ultra networking nerd _**and**_ pop culture maven. @miri64, truly you grace our process! :-)

Reply:
IMHO the question with these kind of things should rather be if it is harmful. Also, for me it is the case that I use this kind of emphasis in written communication to express urgency (which was my intention here). But fine... will remove.